settings/NoticePolicy/NoticePolicyForm.js

```diff
@@ -1,9 +1,161 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { getFormValues } from 'redux-form';
+
+import stripesForm from '@folio/stripes/form';
+import { stripesShape } from '@folio/stripes/core';
+import {
+  Col,
+  Row,
+  Paneset,
+  ExpandAllButton,
+} from '@folio/stripes/components';
+
+import NoticePolicy from '../Models/NoticePolicy';
+import { DeleteConfirmationModal } from '../components';
+import {
+  HeaderPane,
+  GeneralSection,
+  LoanNoticesSection,
+  FeeFineNoticesSection,
+  RequestNoticesSection,
+} from './components';
 
 class NoticePolicyForm extends React.Component {
+  static propTypes = {
+    stripes: stripesShape.isRequired,
+    pristine: PropTypes.bool,
```

Comment:
Please add isRequired to errors property or defaultProps if this property is not required